src/components/application_manager/src/hmi_state.cc

```diff
@@ -279,6 +283,10 @@ mobile_apis::HMILevel::eType DeactivateHMI::max_hmi_level() const {
   using namespace helpers;
   using namespace mobile_apis;
 
+  if (parent() && HMILevel::INVALID_ENUM == parent()->hmi_level()) {
+    return parent()->hmi_level();
+  }
```

Comment:
```suggestion
  if (HMILevel::INVALID_ENUM == parent_hmi_level()) {
    return parent_hmi_level();
  }
```
This function has a null check for `parent()` when necessary